baseapp/grpcserver.go

```diff
@@ -67,6 +68,10 @@ func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
 			app.logger.Error("failed to set gRPC header", "err", err)
 		}
 
+		if logQueries {
+			app.logger.Info("gRPC query received of type: " + fmt.Sprintf("%#v", req))
```

Comment:
The thought here is, this will only be enabled in the event we are actively debugging something, and have an integrator to send me their info logs is much easier than asking to send massive debug logs and/or filter it themselves.

Happy to change it in this PR for the sdk, will probably keep as info for the fork for now though.

Reply:
i think this is what filter logging was created for or one of the reasons. It would be much cleaner to ask an integrator to turn on debugging and paste "....." in the filter logs. This way they only get the logs you want. Currently this approach you will need to ask for a snapshot or they will need to search for the log along side all the other infos. 

The filter would be to see blocks are being created and this debug log. So you as the debugger would make your life easier. 

This is encroaching on devops territory which there are tools for, we dont want to make the sdk more complicated to run and debug.

Reply:
Np, will change sdk side

Reply:
https://github.com/cosmos/cosmos-sdk/pull/19049/commits/fde2d27dbe7e54a00971332a2beac713151e73c6

Reply:
sorry i meant no config changes, just a debug statement

Reply:
For a node under heavy load like ours (250 rps), this might make the debug logs impossible to grok. I know one can filter but it seems helpful for the logs to be somewhat manageable, and there are rare edge cases where this info is needed. I think a config makes the most sense.

Reply:
I can remove it, I just wanted to explain why I added it as a config in the first place. I am actually not familiar with the debug filtering at all so that's why this wasn't clear to me. Happy to remove it